client/blocks/payment-request/custom-button.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { getStripeServerData } from '../stripe-utils';
+
+export const CustomButton = ( { onButtonClicked } ) => {
+	const {
+		theme = 'dark',
+		height = '44',
+		customLabel = 'Buy now',
```

Comment:
@diegocurbelo - since Kris is AFK, would you be able to make the translation addition? For the aria label - could you add that change as well, or create an issue for it?

Reply:
Excellent catch! Fixed in c23ecd80f939576b9198e9d96d1b43e3e556aa25